app/src/main/java/fr/groggy/racecontrol/tv/ui/channel/ChannelCardPresenter.kt

```diff
@@ -34,15 +36,16 @@ class ChannelCardPresenter: Presenter() {
     }
 
     override fun onBindViewHolder(viewHolder: ViewHolder, item: Any) {
+        val resources = RaceControlTvApplication.context?.resources
         val view = viewHolder.view as ImageCardView
         when(item) {
             is BasicChannelCard -> {
                 val type = item.type
                 view.titleText = when(type) {
-                    Wif -> "Main broadcast"
-                    PitLane -> "Pit lane"
-                    Tracker -> "Tracker"
-                    Data -> "Data"
+                    Wif -> resources?.getString(R.string.main_broadcast)
+                    PitLane -> resources?.getString(R.string.pitlane)
+                    Tracker -> resources?.getString(R.string.tracker)
+                    Data -> resources?.getString(R.string.data)
```

Comment:
Please remove this whole translation step here, it is nice I know but it introduce a tech debt at this stage that if for some reason this changes then it doesn't work anymore, if we had control over the api it'd be fine but as this is not the case...

Reply:
Do you mind explaining a bit more on how this introduces technical debt? Because of the dependency on the item.type which is defined by the API?

Reply:
Exactly. If something new gets introduced later on we have to update (currently 3 translations) + we always need to know the type...

Reply:
New would not be a big problem, but if they would remove something, it would also break the original code...
I want to keep it in my own fork, but will remove it from this pull request.